src/nrc/api/filters.py

```diff
@@ -198,8 +196,14 @@ def evaluate(self, event):
             return False
 
         event_data = event.forwarded_msg
-        event_attribute = self._get_event_attribute(event)
-        return event_data[event_attribute] == self.node["value"]
+
+        for key_node in self.node.keys():
+            event_attr = self._get_event_attribute(event, key_node)
+
+            if not event_data[event_attr] == self.node.get(event_attr, {}):
```

Comment:
De `.get(event_attr, {})` is hier niet nodig, je loopt hier namelijk al over de `keys` in de filter heen (en weet dus dat de `key` aanwezig is)